draft-davies-internal-tld.mkd

```diff
@@ -33,51 +37,80 @@ informative:
     title: "Identification of a top-level domain for private use"
     target: https://itp.cdn.icann.org/en/files/root-system/identification-tld-private-use-24-01-2024-en.pdf
     date: 2024-01
+  ICANN-Board-Resolution:
+    title: "Reserving .INTERNAL for Private-Use Applications"
+    target: https://www.icann.org/en/board-activities-and-meetings/materials/approved-resolutions-special-meeting-of-the-icann-board-29-07-2024-en#section2.a
+    date: 2024-07
 
 --- abstract
 
-This document describes the reservation of the "internal" top-level domain for
+This document describes the reservation of the ".internal" top-level domain for
 use in private applications.
 
 --- middle
 
 # Introduction
 
 There are certain circumstances where private network operators may wish to use
-their own domain naming scheme that is not intended to be used or accessible by
-the broader Domain Name System, such as within closed corporate or home networks. 
+their own domain naming scheme that is not intended to be part of the global
+DNS, such as within closed corporate or home networks. 
 
-Long ago, private-use IP address ranges were set aside {{RFC1918}} for
-a similar purpose. IANA nominally acts as a steward for this address space
-which allows for individual network users to use the specified ranges for
-applications that are not intended to be globally routable.
+Private-use IP address ranges were set aside in {{RFC1918}} for a similar
+purpose. {{ICANN-Board-Resolution}} IANA nominally acts as a steward for this
```

Comment:
Not sure why there is a reference to the ICANN Board resolution here, misplaced?

Reply:
Yes, this was erroneous. Thanks for deleting.